lightning/src/ln/channelmanager.rs

```diff
@@ -9559,13 +9569,19 @@ where
 							None
 						}
 					},
+				};
+
+				match (responder, response) {
+					(Some(responder), Some(response)) => return responder.respond(response),
+					_ => return ResponseInstruction::NoResponse
 				}
 			},
+
 			OffersMessage::InvoiceError(invoice_error) => {
 				log_trace!(self.logger, "Received invoice_error: {}", invoice_error);
-				None
+				return ResponseInstruction::NoResponse
 			},
-		}
+		};
```

Comment:
semi-colon not needed

Reply:
Cleaned up in [77c393a](https://github.com/lightningdevkit/rust-lightning/commit/77c393a6eec06158cbe5fe33608fbe00e2386135)
Thanks!